PIL/JpegImagePlugin.py

```diff
@@ -593,7 +593,7 @@ def _save(im, fp, filename):
 
     info = im.encoderinfo
 
-    dpi = info.get("dpi", (0, 0))
+    dpi = [int(x) for x in info.get("dpi", (0, 0))]
```

Comment:
So the rounding to the nearest integer that I'm already doing? Apologies if confused.